### PR TITLE
More classes to hooks

### PR DESCRIPTION
**Pull request by RDIL**
I'm going to do more of this later.  

Signed-off-by: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

**Comment by RDIL** [comment]
@fkorotkov any idea why the pod failed?

**Review by fkorotkov — COMMENTED** [review]
Clickin on Settings doesn't work for me on your fork: nothing happens

**Comment by RDIL** [comment]
I'll investigate.

**Comment by RDIL** [comment]
Fixed.

**Comment by RDIL** [comment]
@fkorotkov review ping.

File: src/components/chips/BuildCreatedChip.tsx [hunk 1]
```diff
@@ -17,37 +17,41 @@ interface Props extends WithTheme {
...
+  const durationInSeconds = Math.floor(durationAgoInSeconds);
+
+  return (
+    <Tooltip
+      title={`Created at ${new Date(creationTimestamp).toLocaleTimeString()} on ${new Date(
+        creationTimestamp,
+      ).toDateString()}`}
+    >
+      <Chip
+        className={this.props.className}
```
Comment (fkorotkov) [inline src/components/chips/BuildCreatedChip.tsx:44]:
```suggestion
        className={props.className}
```

**Review by fkorotkov — CHANGES_REQUESTED** [review]

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**